<?xml version="1.0" ?>
<sdf version="1.4">
  <world name="default">
    <include>
      <uri>model://ground_plane</uri>
    </include>
    <include>
      <uri>model://sun</uri>
    </include>
    <model name="joint14_model">
      <pose>0 0 0.5 0 0 0</pose>
      <link name="body1">
      <pose>0 0 0.5 0 0 0</pose>
        <collision name="geom">
          <geometry>
            <box>
              <size>1 1 1</size>
            </box>
          </geometry>
        </collision>
        <visual name="visual">
          <geometry>
            <box>
              <size>1 1 1</size>
            </box>
          </geometry>
        </visual>
      </link>
      <link name="body2">
      <pose>0 0.5 2 0 0 0</pose>
        <collision name="geom">
          <geometry>
            <box>
              <size>1 1 1</size>
            </box>
          </geometry>
        </collision>
        <visual name="visual">
          <geometry>
            <box>
              <size>1 1 1</size>
            </box>
          </geometry>
        </visual>
      </link>
      <joint type="revolute" name="joint14_revolute_joint">
        <pose>0 0 1 0 0 0</pose>
        <parent>
          <link_name>body2</link_name>
        </parent>
        <child>
          <link_name>body1</link_name>
        </child>
        <axis>
          <xyz>0 0 1</xyz>
        </axis>
      </joint>
    </model>
  </world>
</sdf>
